lib/http/timeout/null.rb

```diff
@@ -46,6 +46,28 @@ def write(data)
         @socket << data
       end
 
+      # Retry reading
+      def __rescue_readable(&block)
```

Comment:
Since the block is a required argument (the only argument, and it's required), I would recommend including it in the method signature.  That said, it'd work just fine as you describe, @zanker .

@tarcieri @zanker : Let me know what you'd prefer and I'll change/not-change accordingly :)

Thanks for this library existing! :+1: